modules/services/sxhkd.nix

```diff
@@ -57,44 +57,19 @@ in
           i3-msg {workspace,move container to workspace} {1-10}
       '';
     };
-
-    extraPath = mkOption {
```

Comment:
I think we should add a `mkRemovedOptionModule` for this. (There are a few examples of usage throughout the repo.)

Reply:
Added